sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/component.py

```diff
@@ -182,7 +184,7 @@ def display_name(self) -> str:
         return self._display_name
 
     @display_name.setter
-    def display_name(self, custom_display_name: str):
+    def display_name(self, custom_display_name: str) -> None:
```

Comment:
Wrong return type

Reply:
Return type on a setter is None. What are you suggesting it should be?